packages/oas-normalize/test/__fixtures__/bundle/definition.json

```diff
@@ -29,12 +29,12 @@
         "content": {
           "application/json": {
             "schema": {
-              "$ref": "./test/__fixtures__/bundle/pet.json"
+              "$ref": "./packages/oas-normalize/test/__fixtures__/bundle/pet.json"
```

Comment:
i mentioned this in the PR description — not sure which is better

Reply:
can probably change this file from `definition.json` to `definition.js` and have it export the def + change this ref to use `require.resolve('oas-normalize/test/__fixtures__/bundle/pet.json')`. wonder if that'd work. it wouldn't be a relative import anymore but i don't think that matters too much since relative file refs are tested in `json-schema-ref-parse` and `openapi-parser` anyways.